ZupIT/ritchie-cli
# Credential for adding Private Repo

### Description

Now the command `rit add repo` has a specific credential, so that the user does not need to regenerate his private tokens each time the command is executed.

### How to verify it
In its first execution, when adding a private repository, the command will request the token, as it already did. However, the command now saves this token in a credential.
After executing the `rit add repo` command adding a new private repository, use the `rit list credential` command, to verify that a new credential has been created in the current environment.

### Changelog

Save and reuse repo credentials for private repos
